packages/rn-tester/js/examples/AnimatedGratuitousApp/AnExSet.js

```diff
@@ -139,10 +139,10 @@ const styles = StyleSheet.create({
   headerText: {
     fontSize: 25,
     color: 'white',
-    shadowRadius: 3,
-    shadowColor: 'black',
-    shadowOpacity: 1,
-    shadowOffset: {height: 1},
+    textShadowRadius: 3,
+    textShadowColor: 'black',
+    textShadowOpacity: 1,
+    textShadowOffset: {height: 1},
```

Comment:
Can you mark this with `// TODO(macOS tags)` and also reference a Github issue to upstream this fix, for tracking purposes?